lib/collector/remote-method.js

```diff
@@ -88,7 +90,32 @@ RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
       if (err) {
         return callback(err)
       }
-      this._post(serialized, nrHeaders, callback)
+      this._post(
+        serialized,
+        nrHeaders,
+        function onResponse(err, res) {
+          if (err) {
```

Comment:
I'm not sure you want to do this. if the request to collector endpoint fails we currently aren't recording those metrics, am I reading that correctly?

Reply:
That's true. I can modify it to be more durable.